app/controllers/concerns/api/taxonomy_scope.rb

```diff
@@ -1,26 +0,0 @@
-module Api
-  module TaxonomyScope
```

Comment:
GIven the wide usage of this module in the plugins, wouldn't make sense to keep it around for a release, just to print a deprecation warning?

Reply:
yeah I was thinking the same when I opened a PR against REX yesterday... I suppose we should keep it as noop?

Reply:
Yes, noop with deprecation message in `included` block